src/cmd/hotspots/mod.rs

```diff
@@ -7,15 +7,39 @@ use prettytable::{format, Table};
 use serde_json::json;
 use structopt::StructOpt;
 
+pub mod transfer;
+
+#[derive(Debug, StructOpt)]
+pub enum Cmd {
+    List(List),
+    Transfer(transfer::Transfer),
+}
+
+impl Cmd {
+    pub fn run(self, opts: Opts) -> Result {
+        match self {
+            Self::List(list) => list.run(opts),
+            Self::Transfer(list) => list.run(opts),
+        }
+    }
+}
+
 /// Get the hotspots for a wallet
 #[derive(Debug, StructOpt)]
-pub struct Cmd {
+pub struct List {
     /// Addresses to get hotspots for
     #[structopt(short = "a", long = "address")]
     addresses: Vec<String>,
+
+    /// Base64 encoded transaction to sign. If no transaction if given
+    /// stdin is read for the transaction. Note that the stdin feature
+    /// only works if the wallet password is set in the
+    /// HELIUM_WALLET_PASSWORD environment variable
+    #[structopt(name = "TRANSACTION")]
```

Comment:
how is this related to the List command?

Reply:
woopsy. total mistake. good catch